projects/media-viewer/src/lib/annotations/comment-set/comment-set-header/comment-set-header.component.html

```diff
@@ -0,0 +1,8 @@
+<div class="commentSummaryHeader">
```

Comment:
not really an issue but i think the ngif should be here

Reply:
i think when it starts to grow with adding search and other subcomponents we gonna have a better pix when it will live